src/components/WithdrawPage.tsx

```diff
@@ -82,7 +82,7 @@ const WithdrawPage = (props: Props): ReactElement => {
   const noShare = !myShareData || myShareData.lpTokenBalance.eq(Zero)
 
   return (
-    <div className={"withdraw " + classNames({ noShare: noShare })}>
+    <div className={classNames({ noShare: noShare, withdraw: true })}>
```

Comment:
since the `noShare` class has been commented out of the scss, we can remove it here. The result is just `<div className="withdraw">`